docs/docsite/rst/scenario_guides/guide_vultr.rst

```diff
@@ -0,0 +1,130 @@
+Vultr
+=====
+
+Ansible offers a set of modules to interact with `Vultr <https://www.vultr.com>`_ cloud platform.
+
+This set of module forms a framework that allows one to easily manage and orchestrate one's infratructure on Vultr cloud platform.
+
+
+Requirements
+------------
+
+There is actually no technical requirement; simply an already created Vultr account.
+
+
+Configuration
+-------------
+
+Vultr modules offer a rather flexible way with regard to configuration.
+
+Configuration is read in that order:
+
+  - Environment Variables (eg. `VULTR_API_KEY`, `VULTR_API_TIMEOUT`)
```

Comment:
Remove leading spaces before `-`

For inline code in RST it's double backticks

```
Environment Variables (eg. ``VULTR_API_KEY``, ``VULTR_API_TIMEOUT``)

```